lib/matplotlib/testing/compare.py

```diff
@@ -367,12 +367,7 @@ def calculate_rms(expectedImage, actualImage):
         raise ImageComparisonFailure(
             "Image sizes do not match expected size: {0} "
             "actual size {1}".format(expectedImage.shape, actualImage.shape))
-    num_values = expectedImage.size
-    abs_diff_image = abs(expectedImage - actualImage)
-    histogram = np.bincount(abs_diff_image.ravel(), minlength=256)
-    sum_of_squares = np.sum(histogram * np.arange(len(histogram)) ** 2)
-    rms = np.sqrt(float(sum_of_squares) / num_values)
-    return rms
+    return ((expectedImage - actualImage) ** 2).mean()
```

Comment:
Huh, I may have missed something but this is "Mean Square" rather than "Root Mean Square", is it not? Is it not missing a call to `np.sqrt`?

Reply:
The previous method (going through the histogram) seems extra complex, why was it done that way?  Is this just a weird historical artifact of supporting numarray/numeric or an attempt to get performance?

Reply:
the missing sqrt is just a typo, fixed.
pretty sure performance of this specific part of the code is irrelevant compared to rendering the png or shelling out to inkscape and ghostscript...